packages/amplication-client/src/Application/code-view/CodeViewExplorerTree.tsx

```diff
@@ -0,0 +1,120 @@
+import { Icon, TreeView } from "@amplication/design-system";
+import React, { useCallback, useEffect, useState } from "react";
+import { Build } from "../../models";
+import { FileDetails } from "./CodeViewPage";
+import { FileExplorerNode } from "./FileExplorerNode";
+import { NodeTypeEnum } from "./NodeTypeEnum";
+import { StorageBaseAxios, StorageResponseType } from "./StorageBaseAxios";
+import { useQuery } from "react-query";
+import { AxiosError } from "axios";
+import { remove } from "lodash";
+import { FileMeta } from "./CodeViewExplorer";
+import "./CodeViewBar.scss";
+
+const CLASS_NAME = "code-view-bar";
+
+type Props = {
+  selectedBuild: Build;
+  onFileSelected: (selectedFile: FileDetails) => void;
+};
+
+const NO_FILES_MESSAGE = "There are no available files to show for this build";
```

Comment:
this is a general comment. I think we should address texts in a different way and not at the end point component. it will give us the ability to handle languages as well as dynamic text change

Reply:
I agree, but it will require a major change. There is already an open issue about localization of text.